overrides/config-overrides/normal/betterquesting/DefaultQuests.json

```diff
@@ -31913,7 +31913,7 @@
           "partialMatch:1": 1,
           "requiredItems:9": {
             "0:10": {
-              "Count:3": 1,
```

Comment:
What happened here?

Reply:
Huh, it looks like the ruridit in this block was 1 and not 64 like everything else. Didn't notice when updating them.

![image](https://github.com/Nomi-CEu/Nomi-CEu/assets/5544386/2e68cc36-cc48-4e0a-b7e4-a945c63bd8c3)

Should we change it back to 1 or keep it at 64 to match the other quests here?

Reply:
Yeah, lets leave it at 64.